rfd/0169-auto-updates-linux-agents.md

```diff
@@ -454,23 +512,22 @@ spec:
   agent_schedules:
     regular:
     - name: default
-      days: ["*"]
+      days: ["*"] # TODO: restrict to work week? Minus Friday?
```

Comment:
Was assuming the restriction would be a Cloud-only setting, and self-hosted customers would have full control.